<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMatrixDialog</class>
 <widget class="QDialog" name="AddMatrixDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rotation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Translation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Scale</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ortho</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Frustum</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>label1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>label2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>label3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>label4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>label5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>label6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-999.990000000000009</double>
       </property>
       <property name="maximum">
        <double>999.990000000000009</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>string</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddMatrixDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddMatrixDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
